test/client/js/libs/handlebars/friendlydateTest.js

```diff
@@ -0,0 +1,24 @@
+"use strict";
+
+const expect = require("chai").expect;
+const moment = require("moment");
+const friendlydate = require("../../../../../client/js/libs/handlebars/friendlydate");
+
+describe("friendlydate Handlebars helper", () => {
+	it("should render 'Today' as a human-friendly date", () => {
+		const time = new Date().getTime();
+		expect(friendlydate(time)).to.equal("Today");
+	});
+
+	it("should render 'Yesterday' as a human-friendly date", () => {
+		const time = new Date().getTime() - 24 * 3600 * 1000;
+		expect(friendlydate(time)).to.equal("Yesterday");
+	});
+
+	it("should not render any friendly dates prior to the day before", () => {
+		[2, 7, 30, 365, 1000].forEach(day => {
+			const time = new Date().getTime() - 24 * 3600 * 1000 * day;
+			expect(friendlydate(time)).to.equal(moment(time).format("L"));
```

Comment:
I thought about that but that depends on locale, which is already causing issues in https://github.com/thelounge/lounge/blob/master/test/client/js/libs/handlebars/localetimeTest.js. I would need to look into mocking the locale, etc. and I won't have time for this on this PR.
I can either let it as is and someone can look into that post-merge, or remove the test, or close the PR. Obviously I'm advocating for the first option :)